ha/util/ha_utils.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import xml.etree.ElementTree as ET
+
+from cortx.utils.log import Log
+
+from ha import const
+from ha.execute import SimpleCommand
+
+
+class HaUtils:
+
+    def __init__(self):
+        self.process = SimpleCommand()
+
+    def get_online_nodes(self):
+        """
+        Get list of online nodes ids.
+        """
+        try:
+            online_nodes_xml = self.process.run_cmd(const.GET_ONLINE_NODES_CMD)
+            # create element tree object
```

Comment:
why using crm and not cluster manager

Reply:
As this code is also getting used in the node alert monitor, we need this to get current node id and name from pacemaker alert script i.e. pcmk_alert